pkg/version/shared_test.go

```diff
@@ -277,6 +290,22 @@ func TestCompare(t *testing.T) {
 	}
 }
 
+func TestClone(t *testing.T) {
+	v1 := parseOrFatalGeneric(t, "1.2")
+	v2 := v1.Clone()
+
+	v1.Decimal[0] = decimal.New(0, 0)
+	assert.NotEqual(t, v2.Decimal, v1.Decimal, "cloned version has different Decimal slice than original")
+}
```

Comment:
How about another test for `Clone`, to verify that the clone and the original are equivalent?

Reply:
Done.